portal-web/test/functional/com/liferay/portalweb/macros/FormsAdminNavigator.macro

```diff
@@ -200,4 +200,27 @@ definition {
 		Navigator.openSpecificURL(url = "${publishFormURL}");
 	}
 
+	macro setLocalNetworkDataProvider {
+		Navigator.openURL();
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Control Panel",
+			portlet = "System Settings");
+
+		SystemSettings.gotoConfiguration(
+			configurationCategory = "Data Providers",
+			configurationName = "Data Providers",
+			configurationScope = "Site Scope");
+
+		WaitForElementPresent(
+			key_checkboxOption = "Access Local Network",
+			locator1 = "Checkbox#ENABLE_CHECKBOX_VIA_SPAN");
+
+		SystemSettings.configureSystemSetting(
+			enableSetting = "${setValue}",
+			settingFieldName = "Access Local Network");
+			
+	}
+
```

Comment:
@Jamersonze Why didn’t you use the existing `FormsAdminNavigator.enableLocalNetworkDataProvider();`?

Reply:
At the time i made the test, the page clicked twice on the save button. Then i realized that the twice clicking was because the macro clicked save/update and then the testcase was saving. Now i realize it was useless.